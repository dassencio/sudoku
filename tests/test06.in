3 3
_ 7 _ _ _ 6 _ 1 _
_ _ 6 3 _ _ 9 _ _
1 _ _ _ 8 _ _ _ 3
9 _ _ 5 _ _ _ 3 _
_ _ 7 _ 2 _ 1 _ _
_ 3 _ _ _ 9 _ _ 4
8 _ _ _ 3 _ _ _ 6
_ _ 9 _ _ 5 3 _ _
_ 4 _ 8 _ _ _ 2 _
