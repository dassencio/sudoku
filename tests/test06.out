3 7 2 9 4 6 8 1 5
4 8 6 3 5 1 9 7 2
1 9 5 7 8 2 4 6 3
9 1 4 5 6 8 2 3 7
5 6 7 4 2 3 1 8 9
2 3 8 1 7 9 6 5 4
8 5 1 2 3 4 7 9 6
7 2 9 6 1 5 3 4 8
6 4 3 8 9 7 5 2 1
